packages/experiments/src/components/Button/Button.view.tsx

```diff
@@ -19,13 +33,56 @@ export const ButtonView: IButtonComponent['view'] = props => {
     icon: Icon,
     content: Text,
     menu: ContextualMenu,
-    menuIcon: Icon
+    menuIcon: Icon,
+
+    primaryActionContainer: Stack,
+    secondaryActionContainer: 'span',
+    splitDivider: 'span'
   });
 
+  if (split) {
+    return (
+      <Slots.root
+        type="button" // stack doesn't take in native button props
+        role="button"
+        onKeyDown={onAltDownKeyDown}
+        {...buttonProps}
+        aria-disabled={disabled}
+      >
+        <Slots.stack horizontal as="span" gap={8} verticalAlign="stretch" horizontalAlign="center" verticalFill>
+          <Slots.primaryActionContainer
```

Comment:
Is all of the content from lines 45-52 same as starting on line 80? May want to avoid the duplication and just render the inner content specially.

Reply:
It's not the same. If you check carefully, the `onClick` is not present in `Slots.root` in lines 45-52, but rather in `Slots.primaryActionContainer` in lines 53-60. On the other hand, `onClick` is present in `Slots.root` starting on line 81.

Reply:
Couldn't you define a `Slots.root` prop that's like `onClick = split ? onClick : undefined`?